### DISCUSSION TITLE
How to list history only for current tab.

### DISCUSSION THREAD

**Discussion by RFDAJE** [gdh1995/vimium-c#1227] [Q&A] [answered]
For Example, I' m visiting Site A in a tab, click through B -> C -> D -> E -> F (last), I can hit `H` to goBack one step to E, but if I want go back to B or C, I have to hit multiple times of `H`, do we have a command to view i.e. list history for current tab? Thanks

**Comment by gdh1995** [comment, marked as answer]
Sorry but browsers including Chrome and Firefox haven't provided APIs to give such information to web-extensions like Vimium C.
